app/controllers/dreams_controller.rb

```diff
@@ -51,8 +51,16 @@ def retreave_tags(hash)
 
   def create_themes(tag_words, dream)
     tag_words.each do | word |
-      tag = Tag.find_or_create_by(word: word)
-      Theme.create(tag: tag, dream: dream)
+      if (tag = Tag.find_by(word: word))
+        Theme.create(tag: tag, dream: dream)
+        if (!tag.appears_more_than_once)
+          tag.appears_more_than_once = true
+          tag.save!
```

Comment:
@will-sklenars 

ok actually i think i understand

this boolean means like, 'there are multiple themes with this tag'

is that it?